sympy/utilities/lambdify.py

```diff
@@ -563,6 +585,7 @@ def isiter(l):
             ','.join(dum_args), lstr, flat, iter_args)
         if len(re.findall(r'\b%s\b' % flat, rv)) > 1:
             raise ValueError('the name %s is reserved by lambdastr' % flat)
+        print (rv)
```

Comment:
Is this for debugging?

Reply:
Sorry for this inconvenience.I will make a pull request again soon after correcting all these things.

Reply:
I would take care of all these things in future. I have updated now the comment. Can I get a review on my logic?